src/mlbstandings/app.py

```
@@ -0,0 +1,52 @@
+    # How did I update scopes for the cloud function esp auth/drive ?
+    scopes = ['https://www.googleapis.com/auth/drive',  # to create spreadsheets
+              'https://www.googleapis.com/auth/spreadsheets',
+              'https://www.googleapis.com/auth/drive.metadata.readonly']  # unlikely we need this?
+    creds = google.auth.default(scopes=scopes)[0]  # type: ignore
+    authed_session = AuthorizedSession(creds)  # type: ignore
+    files = light_google_wrappers.Files(authed_session)
+    sheets = light_google_wrappers.Spreadsheets(authed_session)
+    base_web = web.Web()
+    w = AbstractRateLimitedWeb(base_web, SimpleRateLimiter(15))
+    u = updater.Updater(d, files, sheets, CONTENTS_SPREADSHEET_ID, w)
+    # TODO remove once everything works with new versions
```

Pylint (3.3.7)

[warning] TODO remove once everything works with new versions

(W0511)